commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/impl/PacketReaderImpl.java

```diff
@@ -277,8 +277,8 @@ public BiometricRecord getBiometric(String id, String biometricFieldName, List<S
 			BIR bir = CbeffValidator.getBIRFromXML(IOUtils.toByteArray(biometrics));
 			biometricRecord = new BiometricRecord();
 			if(bir.getOthers() != null) {
-				Map<String, String> others = new HashMap<>();
-				bir.getOthers().forEach(e -> {
+				HashMap<String, String> others = new HashMap<>();
+				bir.getOthers().entrySet().forEach(e -> {
```

Comment:
no null check?

Reply:
Hi Sir, already doing null check in line number 279